index.js

```diff
@@ -0,0 +1,17 @@
+/** @type {import("eslint").Linter.Config}  */
+const defaultConfig = {
```

Comment:
Are all rules from the guide there?

Reply:
@adrianklimek i added ts guide rules for now, will do js guide next